cpp/benchmarks/string/convert_durations.cpp

```diff
@@ -13,25 +13,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-#include <benchmark/benchmark.h>
-
+#include <cudf/column/column_view.hpp>
 #include <cudf/strings/convert/convert_durations.hpp>
 #include <cudf/types.hpp>
+#include <cudf/wrappers/durations.hpp>
 
 #include <cudf_test/base_fixture.hpp>
 #include <cudf_test/column_utilities.hpp>
 #include <cudf_test/column_wrapper.hpp>
 #include <cudf_test/cudf_gtest.hpp>
 
+#include <benchmark/benchmark.h>
+
+#include <benchmarks/fixture/benchmark_fixture.hpp>
+#include <benchmarks/synchronization/synchronization.hpp>
```

Comment:
```suggestion
#include <benchmarks/fixture/benchmark_fixture.hpp>
#include <benchmarks/synchronization/synchronization.hpp>

#include <benchmark/benchmark.h>
```
gbench header is "further" than local headers.

Reply:
gbench headers are not really needed when `benchmarks/fixture/benchmark_fixture.hpp` is included.
some files include it, some don't.
I am creating a cleanup PR for this.

Reply:
@karthikeyann How do you want to do the merge order? Should this PR go first? If so, can you approve?

Reply:
this one should go  first